DATA_PREPARATION.md

```diff
@@ -200,15 +200,21 @@ For `key_value_pair` dataset, an additional field `schema` is required to define
 
 The `fieldSchema` is a dictionary mapping each field to a dictionary called `FieldSchema`. In above example, there are two fields: `answer` and `rationale`. Below is the definition of `FieldSchema`:
```

Comment:
"The `fieldSchema` is a dictionary mapping each field to a dictionary called `FieldSchema`"

Wording a little bit confusing